manifests/apps/jupyter/jupyter-docs.yaml

```diff
@@ -46,7 +46,7 @@ spec:
 apiVersion: dashboard.opendatahub.io/v1
 kind: OdhDocument
 metadata:
-  name: jupyterhub-view-installed-packages
+  name: jupyter-view-installed-packages
```

Comment:
there are a few access.redhat downstream / commercial products links, here, too

@andrewballantyne @DaoDaoNoCode What do you think of the topic of access.redhat.com links in the files here? There are similarities, for sure, in the content behind the pages, but there are also differences, albeit slight.

Reply:
@MelissaFlinn in the main body of juypter-docs.yaml, there are access.redhat.com links to RHOSDS, too. I posted a question in odh.io slack support channel, unsure ...